tslint.json

```diff
@@ -127,7 +127,7 @@
         ],
         "jsdoc-format": true,
         "max-classes-per-file": [  // we generally recommend making one public class per file 
-            true,
+            false,
```

Comment:
This limit was always an arbitrary starting point. It passed by one with all the custom EventArgs and union types made in container.ts.